definitions/json/CaseEvent/CaseEvent.json

```diff
@@ -346,6 +346,20 @@
     "CallBackURLAboutToSubmitEvent": "${ET_COS_URL}/uploadDocument/aboutToSubmit",
     "ShowSummary": "N"
   },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "manageDocUpload",
+    "Name": "Manage Doc Upload",
```

Comment:
It might be worth confirming with Dan what the event should be called. I think this event is better being called Add Documents rather than managing given what it is doing

Reply:
event id renamed